reactor-aeron-core/src/main/java/reactor/aeron/DefaultAeronOutbound.java

```diff
@@ -21,16 +18,10 @@
   }
 
   @Override
-  public AeronOutbound send(Publisher<? extends ByteBuffer> dataStream) {
+  public AeronOutbound send(Publisher<?> dataStream) {
```

Comment:
maybe use some interface simular to Payload interface of Frame interface?

Reply:
Fixed